tests/python/gaia-ui-tests/gaiatest/form_controls/form_control.py

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, Wait
+from gaiatest.apps.base import PageRegion
+
+
+class FormControl(PageRegion):
+    def __init__(self, marionette, locator):
+        element = Wait(marionette).until(expected.element_present(*locator))
+        PageRegion.__init__(self, marionette, element)
```

Comment:
After taking a closer look at it, this might be a big task to deal with. I'd rather to keep this PR not too big. I filed this [follow up bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1203527).